synapse/lib/storm.py

```diff
@@ -441,6 +441,8 @@ def __init__(self, **opts):
         self.setOperFunc('fromtags', self._stormOperFromTags)
         self.setOperFunc('jointags', self._stormOperJoinTags)
 
+        self.setOperFunc('get:tasks', self._stormOperGetTasks)
```

Comment:
I'm not beholden to the macro name `get:tasks()` and am open to other suggestions.